CODE/Logicytics.py

```
@@ -67,35 +74,31 @@ def update() -> tuple[str, str]:
     If the flags are not a tuple, it prints the help message and exits the program.
 
-    Returns:
-        tuple: A tuple containing actions and sub_actions.
     """
     if isinstance(Flag.data(), tuple):
+        global ACTION, SUB_ACTION
         try:
             # Get flags
-            actions, sub_actions = Flag.data()
+            ACTION, SUB_ACTION = Flag.data()
         except Exception:
```

Ruff (0.8.0)

Do not catch blind exception: `Exception`

(BLE001)